XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -195,7 +193,7 @@ private void EnabledRow(Host host, UpdateType type, int index)
                 }
 
                 //check updgrade sequences
-                Updates.UpgradeSequence us = Updates.GetUpgradeSequence(host.Connection, AutoDownloadedXenServerVersions);
+                Updates.UpgradeSequence us = Updates.GetUpgradeSequence(host.Connection, Updates.XenServerVersions);

```

Comment:
I think we should use Updates.GetUpgradeSequence(conn) here, as we do in the other 2 places in the wizard. And we can even get rid of the GetUpgradeSequence(conn, xsVersions) function as it won't be used anymore.